tests/test_anemoi.py

```diff
@@ -0,0 +1,77 @@
+import sys
+from pathlib import Path
+import numpy as np
+from torch.utils.data import DataLoader
+
+# Add the project root to Python path so we can import from graph_weather
```

Comment:
You shouldn't need these lines, if you run `pytest .` from the root of the repo, it'll run these tests and be able to import from graph_weather. So I would remove these lines, and just remember to run the tests from the root. The CI does that for example.